gui/locales/fr.json

```diff
@@ -169,7 +169,7 @@
   "Unlinked Your device is no longer registered on your Cozy.": "Votre ordinateur n’est plus enregistré sur votre Cozy.",
   "Update": "Mettre à jour",
   "Updater Checking for Update": "Recherche de mise à jour…",
-  "Updater Downloading": "Une mise à jour est disponible",
+  "Updater Downloading": "Mise à jour de Cozy Drive",
```

Comment:
I `yarn build:tx` to fetch french translation, I noticed that some translations exist in transifex but not in our repository.